src/locale/locales/de/messages.po

```diff
@@ -599,19 +565,19 @@ msgstr "Meine E-Mail ändern"
 
 #: src/view/screens/Settings/index.tsx:732
 msgid "Change password"
-msgstr ""
+msgstr "Passwort ändern"
 
 #: src/view/screens/Settings/index.tsx:741
 msgid "Change Password"
-msgstr ""
+msgstr "Passwort ändern"
```

Comment:
Again, should we honor the lowercase here?

```suggestion
msgstr "Passwort Ändern"
```

Reply:
I'll capitalise this as well